adoc/admin-software-installation.adoc

```diff
@@ -98,18 +98,32 @@ sudo zypper in ceph-common xfsprogs
 [[helm-tiller-install]]
 === Installing Helm
 
-As of {productname} {productversion},, Helm is part of the {productname} package repository, so to use this,
+As of {productname} {productversion}, Helm is part of the {productname} package repository, so to use this,
 you only need to run the following command from the location where you normally run `skuba` commands:
 
 [source,bash]
 ----
 sudo zypper install helm
 ----
 
+Helm 2 is the default for {productname} 5.0.0. Helm 3 is offered as an alternate tool and may be installed in parallel to aid migration. Helm 3 may be selected by installing from zypper then selecting the alternative.
+
+[source,bash]
+----
+sudo zypper install helm3
+sudo update-alternatives --set helm /usr/bin/helm3
+----
+
+[WARNING]
+Helm 2 is planned to end support in November 2020. Helm 3 is offered as an alternative in {productname} 5.0 and will become the default tool in the following release. For any upgraded installations, please follow the Migration Guide as soon as feasible.
```

Comment:
Which is a suitable version for describes v5? `5.0` or `5.0.0`?

Reply:
We can just use 5, without any further specification.